conftest.py

```diff
@@ -96,10 +103,19 @@ def setup(request):
              "--motd=" + cwd + "/miniircd/testmotd"],
             bg=True)
         miniircd_procs.append(miniircd_proc)
+
+    # determine bitcoind version
+    bitcoind_version_string = subprocess.check_output([bitcoin_path + "bitcoind", "-version"]).split(b'\n')[0]
+    bitcoind_version = get_bitcoind_version(bitcoind_version_string)
+
     #start up regtest blockchain
-    btc_proc = subprocess.call([bitcoin_path + "bitcoind", "-regtest",
-                                "-daemon", "-conf=" + bitcoin_conf])
-    time.sleep(3)
+    bitcoin_args = ["-regtest", "-daemon", "-conf=" + bitcoin_conf]
+
+    if bitcoind_version[1] >= 19:
+        bitcoin_args += ['-acceptnonstdtxn']
```

Comment:
Can you explain what this is for? Accepting non standard transactions in versions of bitcoin higher than 0.19? Is it some feature in 0.20+?

Reply:
see https://github.com/JoinMarket-Org/joinmarket-clientserver/issues/477

Reply:
@AdamISZ with respect to #477, adding the `-acceptnonstdtxn` allows non-standard transactions (which was the default in bitcoin core < 0.19) for the regtest network.